lib/validation/schema/specVersion/kind/project.json

```diff
@@ -381,6 +396,120 @@
 				}
 			}
 		},
+		"builder-bundles-bundleDefinition-4.0": {
+			"type": "object",
+			"additionalProperties": false,
+			"required": ["name"],
+			"properties": {
+				"name": {
+					"type": "string"
+				},
+				"defaultFileTypes": {
+					"type": "array",
+					"items": {
+						"type": "string"
+					}
+				},
+				"sections": {
+					"type": "array",
+					"if": {
+						"$comment": "Add async prop only if mode = 'require'",
+						"items": {
+							"type": "object",
+							"mode": {
+								"enum": ["require"]
+							}
+						}
+					},
+					"then": {
+						"items": {
+							"type": "object",
+							"additionalProperties": false,
+							"required": ["mode", "filters"],
+							"properties": {
+								"name": {
+									"type": "string"
+								},
+								"mode": {
+									"enum": ["raw", "preload", "require", "provided", "bundleInfo", "depCache"]
```

Comment:
here, only "require" should be listed, as only then the "async" flag is supported.

Reply:
Fixed with: https://github.com/SAP/ui5-project/pull/751